### PR TITLE
More sorting

### PR DESCRIPTION
**Pull request by Orang-utan**

### CODE REVIEW

**Comment by Orang-utan** [comment]
I added more ways of sorting. So essentially, you can now sort by Date and Percent Paid on the results page. I think most of the changes happened on the Pagenation React file. Attached are some screenshots of Percent Paid sort (ascending and descending).

Screen Shot 2020-06-29 at 10 42 45 AM
Screen Shot 2020-06-29 at 10 43 03 AM

File: app/templates/main/results.html [hunk 1]
```diff
@@ -55,6 +55,35 @@
             {{ f.begin_form(cba, flashes) }}
             {{ f.render_form(cba) }}
             {{ f.end_form(cba) }}
+            <br />
+            <button class="ui button" 
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByDateDesc()">
```
Comment (Orang-utan) [inline app/templates/main/results.html:62]: for this - I made the Result List (a React component) a global variable by attaching it to the Browser window. This means I can directly call on the functions within React component via a button.

File: app/templates/main/results.html [hunk 2]
```diff
@@ -55,6 +55,35 @@
...
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByDateAsce()">
+               Date (Ascending)
+            </button>
+
+            <button class="ui button" 
+            href="/search" 
+            style="font-family: Avenir; width: 80%; margin-bottom: 20px; text-align: center"
+            onclick="window.resultList.sortByPaidDesc()">
```
Comment (Orang-utan) [inline app/templates/main/results.html:76]: for this - I made the Result List (a React component) a global variable by attaching it to the Browser window. This means I can directly call on the functions within React component via a button.

File: app/templates/main/results.html [hunk 3]
```diff
@@ -68,7 +97,7 @@ <h2>
             <br>
               <p style="font-size: 20px; font-weight: 600; text-transform: uppercase; color: #0f4d90">No Results Found</p>
             {% else %}
-            <div id="result-list-target"></div>
+            <div id="result-list-target" page_size="20"></div>
```
Comment (Orang-utan) [inline app/templates/main/results.html:100]: so now you're able to pass in a page_size variable into the React component prop. The result list component will take the page_size and render that number of entries per page.

File: app/templates/main/pagenation/src/index.js [hunk 4]
```diff
@@ -113,11 +122,74 @@ const ResultList = () => {
 class ResultListContainer extends React.Component {
```
Comment (Orang-utan) [inline app/templates/main/pagenation/src/index.js:122]: I wrapped the actual Result List within another class Component (Hooks don't support Ref yet I think).. So all of the sorting functionalities happen below here.

**Comment by caroljli** [comment]
Thanks Daniel! The comments are very helpful.

**Pull request merged**